comms/c3270/Portfile

```
@@ -1,13 +1,13 @@
 PortSystem          1.0
 
 name                c3270
-version             3.3.15ga9
+version             4.3ga4
 revision            3
-set branch          [join [lrange [split ${version} .] 0 1] .]
+set branch          4.3
 categories          comms
 platforms           darwin
 license             BSD
-maintainers         nomaintainer
+maintainers         {icloud.com:willian.nieckarz @sinskinner}
```

Ports should also be marked `openmaintainer` unless there is an explicit reason why you and only you should be able to update this port:

```suggestion
maintainers         {icloud.com:willian.nieckarz @sinskinner} openmaintainer
```